src/components/PopoverMenu/Popover.tsx

```diff
@@ -72,12 +72,7 @@ export default function PopoverDropdown(props: PopoverProps): JSX.Element {
             ...elements,
             ...section.map((item, itemIndex) => {
               return (
-                <MenuItem
-                  onClick={() => onClick(item)}
-                  key={`option-${itemIndex}`}
-                  sx={itemStyles(item.type)}
-                  disableRipple={true}
-                >
+                <MenuItem onClick={() => onClick(item)} key={`option-${itemIndex}`} sx={itemStyles(item.type)} disableRipple={true} disabled={item.disabled}>
```

Comment:
The prettier `printWidth` in this repo is 160, should I update it to 120 to match `terraware-web`?

Reply:
Being consistent sounds good.

Reply:
Cool, `prettier` isn't used in the build flow so I did not opt for a "apply new prettier config to every file" commit. Let me know if you'd like to see that in another PR or if you are OK that it is slowly applied to files that are touched in the future.

Reply:
Fine with what you suggest. In the interest of time, I agree with the as-touched in the future approach.